client/state/selectors/is-two-step-sms-enabled.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Is two-step sms enabled for the current user?
+ *
+ * @param  {Object} state Global state tree
+ * @return {Boolean} return true if two-step sms is enabled
+ */
+export default function isTwoStepSmsEnabled( state ) {
+	return get( state, 'userSettings.settings.two_step_sms_enabled', false );
```

Comment:
It is a safe default in a sense that the `isTwoStepSMSEnabled` method of the `lib/user-settings` objects behaves exactly the same way. So I'm keeping the status quo - it won't break the existing UI after it's migrated to the Redux version.

But returning `null` here is more correct -- the UI has a chance to distinguish between three states (unknown, true, false) if it decides to do so, and the default is still a falsy value. I'll change this.